src/App/Exercises/ReactExercises/LocalDevFetch/Components/Forms/FormsToDo.jsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react';
+import './formsToDo.css';
+import { setterFunction } from '../../HelpFunctions';
+import { ButtonsToDoList } from '../Buttons/ButtonsToDoList';
+
+export const FormsToDo = ({ setIsActiveForms, loadTheData }) => {
+  const [dataTitle, SetDataTitle] = useState('');
+  const [dataAuthor, SetDataAuthor] = useState('');
+  const [dataContext, SetDataContex] = useState('');
+  const [answer, setAsnwer] = useState('');
+  const [erroAnswer, setErrorAnswer] = useState('');
+  const [isError, setIsError] = useState(false);
+
+  const sendDataToApi = async (title, author, context) => {
+    console.log(' Chyba poszlo ');
+    const respons = await fetch('http://localhost:3333/api/todo', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        title: title,
+        note: context,
+        author: author,
+      }),
+    });
```

Comment:
W tym miejscu bym użył też `requestHandlera` 😃  trzeba rozszerzyć tego funkcjonalność, by móc dodać `body` i użyć 😃

Reply:
Ok, dzieki bardzo za zwrócenie uwagi na to . Taki miałem właśnie plan żeby rozszerzyć RequestHandlera. Tylko najpierw chciałem zrobić to łopatologicznie żeby zobaczyć czy dział ; )